mbox/native/pom.xml

```diff
@@ -1,39 +1,148 @@
-                        <source>
-                            <directory>../src/main/cpp</directory>
-                            <includes>
-                                <include>**/*.c</include>
-                            </includes>
-                        </source>
-                        <source>
-                            <directory>../target/native</directory>
-                            <dependencyAnalysisParticipation>
-                                false
-                            </dependencyAnalysisParticipation>
-                        </source>
-                    </sources>
-                </configuration>
-            </plugin>
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!--
+
+    Copyright (c) 1997, 2021 Oracle and/or its affiliates. All rights reserved.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0, which is available at
+    http://www.eclipse.org/legal/epl-2.0.
+
+    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the
+    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+    version 2 with the GNU Classpath Exception, which is available at
+    https://www.gnu.org/software/classpath/license.html.
+
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+			    http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+	<groupId>com.sun.mail</groupId>
+	<artifactId>all</artifactId>
+	<version>2.0.1-SNAPSHOT</version>
+	<relativePath>../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.sun.mail</groupId>
+    <artifactId>libmbox</artifactId>
+    <packaging>so</packaging>
+    <name>Jakarta Mail API mbox native library</name>
+    <description>Jakarta Mail API mbox native library</description>
+    <url>https://github.com/eclipse-ee4j/jakartamail</url>
+
+    <properties>
+		<m64>
+		</m64>
+		<compiler.name>
+		    c89
+		</compiler.name>
+		<compiler.start.options>
+		    ${m64} -Xa -xO2 -v -D_REENTRANT -KPIC
+		    -I${env.JAVA_HOME}/include -I${env.JAVA_HOME}/include/solaris
+		</compiler.start.options>
+		<linker.name>
+		    c89
+		</linker.name>
+		<linker.start.options>
+		    -G ${m64} -KPIC -z text
+		</linker.start.options>
+		<linker.arch>
+		    ${env.MACH}
+		</linker.arch>
+		<linker.libs>${env.JAVA_HOME}/jre/lib/${linker.arch}</linker.libs>
+		<linker.end.options>
+		    -L${linker.libs} -lmail -ljava -lc
+		</linker.end.options>
+    </properties>
+
+    <profiles>
+	   <profile>
+        <!--
+        Override the settings necessary to build with JDK 11.
+        -->
+        <id>11</id>
+        <activation>
+            <jdk>11</jdk>
```

Comment:
this profile is not needed since using JDK11 for build is required

Reply:
I have removed it, thank you.